.eslintrc

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "@ministryofjustice/eslint-config-fb"
+  "extends": "@ministryofjustice/eslint-config-fb",
+  "parserOptions": {
+	  "ecmaVersion": "2019"
```

Comment:
Adds support for object spread...but ideally we use a set of sensible & modern defaults across all projects

Reply:
Indeed - why not update `eslint-config-fb` thusly?

Reply:
https://github.com/ministryofjustice/eslint-config-fb/pull/2